### PR TITLE
Improved wordle

### PR DESCRIPTION
**Pull request by ericpretzel**
Various improvements to the Wordle cog.
- Added /keyboard command - lets the user see what letters they have already used and whether they are correct or not. The message is only viewable by the command invoker to prevent clutter.
- Added **HARD MODE** which is hard. Should be the same as what's on the website. 
- Minor restructuring so it is not as bad, including better (but probably still not very proper) documentation
- Two new words: "pretz" and "tinky." These additions will vastly improve the Wordle experience.

### CODE REVIEW

File: extension/wordle.py [hunk 1]
```diff
@@ -69,28 +75,69 @@ def inner_check(msg: discord.Message):
...
+        game.result = False
+        await ctx.interaction.channel.send(self.quit_message(ctx.author.id))
+        return await ctx.respond('Successfully quit your game.', ephemeral=True)
+    
+    
+    @slash_command(
+        guild_ids=[config.WIDMARK_CLAN_GUILD_ID],
+        description='Shows a QWERTY keyboard of your current wordle letter guesses.'
+    )
+    async def keyboard(self, ctx: discord.ApplicationContext):
```
Comment (erakai) [inline extension/wordle.py]: typing /keyboard will be annoying in mobile give it like /k as an alias

**Review by erakai — COMMENTED** [review]
give alias to ensure client satisfaction, looks good besides. beautiful clean code as always!

**Review by lancemathias — APPROVED** [review]
igtm, but /keyboard is not very user friendly. Maybe you can automatically show the keyboard while the game is active instead.

**Comment by ericpretzel** [comment]
Thanks for the feedback guys! I'll try to make an alias for the keyboard command but I'm not really sure how. Otherwise, I will put it in the original message.

**Comment by RussellChien** [comment]
only noobs can't type out the word keyboard on their phone. but did you make sure to test that the keyboard looks ok on mobile too?

**Comment by ericpretzel** [comment]
I made it so there's now a keyboard. Here's an example of what the keyboard looks like.
image

**Comment by ericpretzel** [comment]
Wait there is an issue like @RussellChien described: it looks horrible on mobile.
image
Any suggestions?

**Comment by ericpretzel** [comment]
I have fixed it for mobile (hopefully) now. One last review please!!!!

**Comment by RussellChien** [comment]
good work. the users of wid bot will be very pleased with this new update

**Pull request merged**